src/Raven.Studio/typescript/models/database/index/indexFieldOptions.ts

```diff
@@ -496,41 +574,65 @@ class indexFieldOptions {
         });
     }
     
-    static defaultFieldOptions(indexHasReduce: KnockoutObservable<boolean>, engineType: KnockoutObservable<Raven.Client.Documents.Indexes.SearchEngineType>) {
-        return new indexFieldOptions(indexFieldOptions.DefaultFieldOptions, indexFieldOptions.getDefaultDto(), indexHasReduce, engineType,
-            indexFieldOptions.globalDefaults(indexHasReduce, engineType));
+    static defaultFieldOptions(indexHasReduce: KnockoutObservable<boolean>, engineType: KnockoutObservable<Raven.Client.Documents.Indexes.SearchEngineType>, indexConfiguration?: Raven.Client.Documents.Indexes.IndexConfiguration,
+                          databaseIndexConfiguration?: Record<string, models.serverWideOnlyEntry | models.databaseEntry<string | number>>) {
```

Comment:
Put `Record<string, models.serverWideOnlyEntry | models.databaseEntry<string | number>>` to named type